### PR TITLE
Adds new mob: Space Laser Beam Carp 

### PR DESCRIPTION
**Pull request by Dorsisdwarf**
Space Laser Beam Carp are introduced as a child mob of spacecarp
 -Attacks with laser beams instead of biting
 -Modified interaction messages
 -Custom-made sprite, edited from current spacecarp images.
 -Currently only possible to encounter via admin intervention
layzorcerp
EDIT: Image was taken before the carp were made to actually fire lasers.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/carp.dm [hunk 1]
```diff
@@ -58,4 +58,23 @@
...
+	name = "Space Laser Beam Carp"
+	desc = "A ferocious, fang-bearing creature that resembles a fish.This one seems angry. Perhaps this is due to the laser gun welded to it's head?"
+	response_disarm = "gently pushes aside the threatening laser barrel"
+	response_help = "carefully pets"
+	var/weapon1
+	ranged = 1
+	rapid = 1
+	retreat_distance = 5
+	minimum_distance = 5
+	weapon1 = /obj/item/weapon/gun/energy/laser/captain
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/carp.dm]: this weapon1 var is irrelevant.
All the weapon var does is make the mob drop this weapon in it's Die() proc, which you've already overriden to just delete the mob.

You can remove the var/weapon1 and the assigning of the captains laser to it
The mob can shoot WITHOUT the need of this weapon var.

**Comment by RemieRichards** [comment]
Once you've done as I've asked then the Code is good to go.

**Comment by Dorsisdwarf** [comment]
Removed the var and the weapon. Thanks for clarifying that it wasn't needed.

**Comment by GunHog** [comment]
You are a monster for making Laser carp. I love it. I hope you make it more than just admin only though. Maybe a prob(5) chance whenever normal carp spawn!

**Comment by RemieRichards** [comment]
Right, Code is solid, I'll try to get someone with Maintainer access to sort this now.

**Comment by Iamgoofball** [comment]
I LOVE YOU

**Pull request merged**